### ISSUE TITLE
man pages are installed in $PREFIX, should be $PREFIX/man/man1

### ISSUE THREAD

**Issue by timsnyder** [conda-forge/scons-feedstock#40]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Looking in my environment, I see scons manpages in the root PREFIX directory.  Those should be installed in `$PREFIX/man/man1` for them to be found correctly and displayed by man.

### Installed packages

```shell
➜ conda list
# packages in environment at /home/t.snyder/miniforge3/envs/scons:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
bzip2                     1.0.8                hd590300_5    conda-forge
ca-certificates           2024.2.2             hbcca054_0    conda-forge
ld_impl_linux-64          2.40                 h41732ed_0    conda-forge
libexpat                  2.6.1                h59595ed_0    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 13.2.0               h807b86a_5    conda-forge
libgomp                   13.2.0               h807b86a_5    conda-forge
libnsl                    2.0.1                hd590300_0    conda-forge
libsqlite                 3.45.1               h2797004_0    conda-forge
libuuid                   2.38.1               h0b41bf4_0    conda-forge
libxcrypt                 4.4.36               hd590300_1    conda-forge
libzlib                   1.2.13               hd590300_5    conda-forge
ncurses                   6.4                  h59595ed_2    conda-forge
openssl                   3.2.1                hd590300_0    conda-forge
pip                       24.0               pyhd8ed1ab_0    conda-forge
python                    3.12.2          hab00c5b_0_cpython    conda-forge
python_abi                3.12                    4_cp312    conda-forge
readline                  8.2                  h8228510_1    conda-forge
scons                     4.6.0.post1     py312h7900ff3_0    conda-forge
setuptools                69.1.1             pyhd8ed1ab_0    conda-forge
tk                        8.6.13          noxft_h4845f30_101    conda-forge
tzdata                    2024a                h0c530f3_0    conda-forge
wheel                     0.42.0             pyhd8ed1ab_0    conda-forge
xz                        5.2.6                h166bdaf_0    conda-forge
```

### Environment info

```shell
➜ conda info

     active environment : scons
    active env location : /home/t.snyder/miniforge3/envs/scons
            shell level : 2
       user config file : /home/t.snyder/.condarc
 populated config files : /home/t.snyder/miniforge3/.condarc
          conda version : 23.11.0
    conda-build version : 3.27.0
         python version : 3.10.12.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=icelake
                          __conda=23.11.0=0
                          __glibc=2.28=0
                          __linux=4.18.0=0
                          __unix=0=0
       base environment : /home/t.snyder/miniforge3  (writable)
      conda av data dir : /home/t.snyder/miniforge3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/t.snyder/miniforge3/pkgs
                          /home/t.snyder/.conda/pkgs
       envs directories : /home/t.snyder/miniforge3/envs
                          /home/t.snyder/.conda/envs
               platform : linux-64
             user-agent : conda/23.11.0 requests/2.31.0 CPython/3.10.12 Linux/4.18.0-305.25.1.el8_4.x86_64 rhel/8.4 glibc/2.28 solver/libmamba conda-libmamba-solver/23.11.1 libmambapy/1.5.6
                UID:GID : 12025:12025
             netrc file : None
           offline mode : False
```

**Comment by kbrindley** [comment]
In v4.8.1 the man pages are not installed at all. I can open a minimal PR for review and discussion.

```
$ conda list scons
# packages in environment at ...
#
# Name                    Version                   Build  Channel
scons                     4.8.1           py312h7900ff3_0    conda-forge
$ find $CONDA_PREFIX -name "scons.1"
```

I don't think this is related to https://github.com/SCons/scons/issues/4577 because this recipe uses the PyPI ``*.tar.gz`` file, which does have the man pages as noted in issue description.

Note that some systems look in ``$CONDA_PREFIX/share/man/man1``, so it may be necessary to duplicate the man pages in more than one location for broader system/OS support.

**Comment by kbrindley** [comment]
https://github.com/conda-forge/scons-feedstock/pull/46

**Comment by stuertz** [comment]
fixed with #46
